app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee.erb

```diff
@@ -32,6 +32,7 @@ ShortFormApplicationService = (
       assistedHousingRentBurden: null
     householdIncome: { incomeTotal: null, incomeTimeframe: 'per_year' }
     groupedHouseholdAddresses: []
+    prioritiesSelected: {}
```

Comment:
@callaghanc after all our rigorous checkbox error testing we missed _one_ case which is right when the application initializes, if `prioritiesSelected` isn't initialized to an empty object then it gets confused when you try to check for the properties `prioritiesSelected.no`, etc. which means that you were able to hit Next without selecting anything (meanwhile JS errors popped up about object being undefined). 😖 

Anyway, I updated this branch, and the review instructions since ashley was already testing the feature last night.